packages/visual-editor/src/utils/schema/getSchema.ts

```diff
@@ -48,18 +43,20 @@ export const getSchema = (data: TemplateRenderProps): Record<string, any> => {
       : getDefaultSchema(document);
 
     // Resolve all fields in the schema markup
-    const resolvedSchema = resolveSchemaJson(document, schemaMarkupJson);
+    const resolvedSchema = resolveSchemaJson(data, schemaMarkupJson);
 
     const parsedSchemaEditorMarkup = removeEmptyValues(resolvedSchema);
     const currentPageUrl = resolveSchemaString(
       document,
-      "https://[[siteDomain]]/[[path]]"
+      document.siteDomain
+        ? "https://[[siteDomain]]/[[path]]"
+        : `${relativePrefixToRoot}[[path]]`
```

Comment:
Is path the result of getPath so relativePrefixToRoot is already incorporated? I can't remember.

Reply:
I guess I'm saying do we even need relativePrefixToRoot then if path has already incorporated it?